client/src/js/modules/runs/router.js

```diff
@@ -0,0 +1,19 @@
+define(['utils/lazyrouter'], function(LazyRouter) {
+    var Router = LazyRouter.extend({
+        appRoutes: {
+            'runs/overview(/bl/:bl)(/:bl)(/run/:run)(/:id)': 'fetchBeamlinesForRuns'
```

Comment:
Think there is an extra (/:bl) in here.

Reply:
Also I think this should be part of the stats module. So you can add this route and controller to the stats module. You would need to change the view filename to avoid conflicts so "runs_overview.js" view.

While it does have a different route prefix '/runs' instead of '/stats' the page is just a wrapper for beamline stats page. So I think it makes sense for it to live in the stats module.